app/controllers/comments_controller.rb

```diff
@@ -95,6 +98,12 @@ def create
     else
       render json: { status: "errors" }
     end
+  rescue Pundit::NotAuthorizedError
```

Comment:
What is the point of rescuing this here only to re-raise it?

Reply:
Because `StandardError` is the ancestor of most exceptions, including `Pundit::NotAuthorizedError`. Liquid tags, as I mentioned in the other PR, should ideally raise a custom exception, not a builtin one that's up top in the exception tree. Since for now they don't by rescueing `StandardError` we end up silencing `Pundit::NotAuthorizedError` breaking the website, since that exception is used by Rails to return an HTTP not authorized error. 

I could have used one rescue statement and checked the type, but I think this is more explicit (and avoids doing a check Ruby already does builtin :D)